### PR TITLE
[staging] Adding missing lang strings for XTD plugins

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/25344
Replaces https://github.com/joomla/joomla-cms/pull/25472
### Summary of Changes
Loading main administrator joomla xx-XX.ini is not a good idea as it may override some frontend xx-XX.ini strings.
The only reason `$lang->load('joomla', JPATH_ADMINISTRATOR);` was added for com_menus and com_modules was to get these strings from backend.
These calls are now deleted.

The correct solution was to add then in frontend.
Some looks weird to add:
```
JTOGGLE_HIDE_SIDEBAR="Hide the sidebar"
JTOGGLE_SHOW_SIDEBAR="Show the sidebar"
```
but as they show as untranslated when using the Field xtd, I added them.
_(it would be interesting to find out why they are needed there and if corrected, delete them in 4.0)_

### Testing Instructions
Set Language debug on and load each xtd plugin when editing an article in frontend.
Check if any string is missing.

### After patch
The main display change with Protostar is the corrected `JGLOBAL_LIST_ALIAS="(<span>Alias</span>: %s)"` string.

@SharkyKZ @Quy

### CODE REVIEW

File: language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -219,6 +231,7 @@ JGLOBAL_FIELD_CREATED_LABEL="Created Date"
 JGLOBAL_FIELD_FEATURED_DESC="Assign the article to the featured blog layout."
 JGLOBAL_FIELD_FEATURED_LABEL="Featured"
 JGLOBAL_FIELD_FIELD_CACHETIME_DESC="The number of minutes before the cache is refreshed."
+JGLOBAL_FIELD_GROUPS="Field Groups"
```
Comment (Quy) [inline language/en-GB/en-GB.ini]: Alpha sort.

File: language/en-GB/en-GB.ini [hunk 2]
```diff
@@ -247,12 +260,16 @@ JGLOBAL_GT="&gt;"
 ; The following string is deprecated and will be removed with 4.0.
 JGLOBAL_HELPREFRESH_BUTTON="Refresh"
 JGLOBAL_HITS="Hits"
+JGLOBAL_HITS_ASC="Hits ascending"
+JGLOBAL_HITS_DESC="Hits descending"
```
Comment (Quy) [inline language/en-GB/en-GB.ini]: Alpha sort.

File: language/en-GB/en-GB.ini [hunk 3]
```diff
@@ -292,7 +312,15 @@ JGLOBAL_YOU_MUST_LOGIN_FIRST="Please login first"
 
 JGRID_HEADING_ACCESS="Access"
 JGRID_HEADING_ID="ID"
+JGRID_HEADING_ACCESS_ASC="Access ascending"
+JGRID_HEADING_ACCESS_DESC="Access descending"
+JGRID_HEADING_ID_ASC="ID ascending"
```
Comment (Quy) [inline language/en-GB/en-GB.ini:317]: Alpha sort.

File: language/en-GB/en-GB.ini (hunk 1 above)
Comment (Quy) [inline language/en-GB/en-GB.ini]: Alpha sort.

File: language/en-GB/en-GB.ini (hunk 2 above)
Comment (infograf768) [reply language/en-GB/en-GB.ini]: Doing now

Comment (SharkyKZ) [reply language/en-GB/en-GB.ini]: Move `JGLOBAL_HITS_DESC` after `JGLOBAL_HITS_COUNT`.

File: language/en-GB/en-GB.ini (hunk 3 above)
Comment (infograf768) [reply language/en-GB/en-GB.ini:317]: doing now

File: language/en-GB/en-GB.ini (hunk 1 above)
Comment (infograf768) [reply language/en-GB/en-GB.ini]: doing now

**Comment by infograf768** [comment]
@Quy Should be OK now. tks.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 659eeca0f2652df53a503749423a8c99dba40025<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25475">issues.joomla.org/tracker/joomla-cms/25475</a>.</sub>

**Comment by infograf768** [comment]
@Quy
Please test again. I found out why 
```
JTOGGLE_HIDE_SIDEBAR
JTOGGLE_SHOW_SIDEBAR
```
was showing as untranslated for the Field xtd plugin.
Sidebar was also loaded for modal which should not have been done. :)
Therefore corrected the view.html.php and took off the strings from frontend.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9442de7a88619df54494d98fd57bbf04a70248ee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25475">issues.joomla.org/tracker/joomla-cms/25475</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 22b09f6786b1fa4d66ce7cc0753bc1c28e7f1d1a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25475">issues.joomla.org/tracker/joomla-cms/25475</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25475">issues.joomla.org/tracker/joomla-cms/25475</a>.</sub>

**Comment by infograf768** [comment]
@HLeithner
Please merge so that TTs can work on it asap.

**Comment by HLeithner** [comment]
thx

**Pull request merged**